src/main/features/claim/views/check-and-send.njk

```diff
@@ -105,6 +105,9 @@
     {% else %}
       {{ row('Interest rate', t('I don’t want to claim interest'), paths.interestPage.uri, bold = true) }}
     {% endif %}
+    {% if helpWithFees %}
```

Comment:
Maybe this is too ambiguous; it's specifically checking if the help with fees Feature is enabled for the service, but perhaps it could be misinterpreted as checking if it's a HWF claim?